### ISSUE TITLE
Build time error happen in DIRECT_XIP, MCUBOOT_DIRECT_XIP_REVERT and MCUBOOT_HW_ROLLBACK_PROT are enabled.

### ISSUE THREAD

**Issue by Sherryzhang2** [mcu-tools/mcuboot#1052]
Build time error happen in DIRECT_XIP, MCUBOOT_DIRECT_XIP_REVERT and MCUBOOT_HW_ROLLBACK_PROT are enabled.

```
mcuboot/boot/bootutil/src/loader.c:2890:9: error: 'slot_usage' undeclared (first use in this function); did you mean 'slot_usage_t'?
     if (slot_usage[BOOT_CURR_IMG(state)].swap_state.image_ok == BOOT_FLAG_SET) {
```

**Comment by d3zd3z** [comment]
@Sherryzhang2: Is this resolved by #1053?

**Comment by Sherryzhang2** [comment]
Yes. It can be closed now.
